docs/reference/configuration.md

```diff
@@ -81,6 +81,25 @@ class MyApp : android.app.Application {
 
 1. From version 1.1.0, you can provide your sample rate value. The value gets evaluated on every new session creation to determine whether the next session is sampled or not. When a session is not sampled, none of its signals are exported. Default value is `1.0`.
 
+### Disk buffering behavior
+
+You can configure how [disk buffering](index.md#disk-buffering) works. For example:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            .setDiskBufferingConfiguration(DiskBufferingConfiguration.enabled()) // <1>
```

Comment:
The example here describes how to enable/disable disk buffering. Do we plan on adding more settings here in the future?